curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc174fcf86c76b9248c6eb2.md

```diff
@@ -7,7 +7,19 @@ dashedName: step-1
 
 # --description--
 
-HTML elements have opening tags like `<h1>` and closing tags like `</h1>`. The text an element will display goes between its opening and closing tags.
+HTML elements have an opening and closing tag with content in between.
+
+Here is the basic syntax:
+
+```html
+<openingTag>content</closingTag>
+```
+
+The first element you will learn about is the `h1` element. The `h1` element is a heading element and is used for the main heading of a webpage.
+
+```html
+<h1>This is a main heading</h1>
+```
```

Comment:
We could remove that first example if the concern is that it might be to long.
But we should beef up this descirption and show at least something more on how to work with h1 elements.
because the data is showing here that people are still guessing on what to due for this first step
https://github.com/freeCodeCamp/freeCodeCamp/issues/54024#issue-2175526604

even with the updates made a few weeks ago, the data really hasn't changed.
So IMO, we should provide at least a little bit more information on how elements work

Reply:
Yeah, this is literally the first step. Tags are new, and it's okay to be long here.